.travis.Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:18.04
 
 RUN apt-get -qq update && \
-    apt-get install -y sudo docker.io git make btrfs-tools libdevmapper-dev libgpgme-dev libostree-dev curl
+    apt-get install -y sudo golint docker.io git make btrfs-tools libdevmapper-dev libgpgme-dev libostree-dev curl
```

Comment:
Is the addition of `golint` necessary? The `Makefile` is still installing that, isn‘t it?

Reply:
Not necessary anymore, thanks!